src/base/bittorrent/session.cpp

```diff
@@ -4545,18 +4549,15 @@ void Session::startUpTorrents()
                     resumeData.useAutoTMM = false;
                     resumeData.savePath = storageLocation;
                     resumeData.downloadPath = {};
-                    LogMsg(tr("Torrent '%1' is assigned the recovered category '%2' whose paths don't match the torrent's path."
-                              " Torrent is switched to \"Manual\" mode.").arg(torrentID.toString(), category), Log::WARNING);
+                    LogMsg(tr("Torrent category paths don't match the save path. Torrent is switched to Manual mode."
```

Comment:
It is important that we are talking about the "recovered" category. In regular case, this is a normal phenomenon, the torrent will simply be moved to the folder according to the category settings. But if the category was recovered, then we could lose its original settings, so in case of a discrepancy with the current torrent paths, we switch them to Manual mode to prevent unwanted data movement and give the user the opportunity to solve it on their own.